### PR TITLE
Change to using FastAPI

### PR DESCRIPTION
**Pull request by NicoEvZ**

### CODE REVIEW

File: web_logger/__init__.py [hunk 1]
```diff
@@ -1,45 +1,38 @@
...
+from .Devices.Arduino import Arduino
+import logging
+app = FastAPI
+
+devices = [Arduino('/dev/ttyUSB0')]
+logging.info("Added arduino to device list")
+
+@app.get("/")
+def home():
+    return FileResponse("home.html")
```
Comment (OkeWoke) [inline web_logger/__init__.py]: This will just return the empty template html, not the rendered template I think.

File: web_logger/__init__.py [hunk 2]
```diff
@@ -1,45 +1,38 @@
...
+@app.get("/Devices")
+def ListDevices():
+    """Gives a list of all devices and their dimensions"""
+    device_dim = {}
+    for dev in devices:
+        device_dim[dev.name] = dev.get_dimensions()
+    return {'Devices': device_dim}
+
+@app.get("/Devices/{device_id}")
+def ListDeviceIDs(device_id: int):
```
Comment (OkeWoke) [inline web_logger/__init__.py:23]: change func name

**Review by OkeWoke — CHANGES_REQUESTED** [review]

File: web_logger/__init__.py [hunk 3]
```diff
@@ -1,45 +1,38 @@
...
-api.add_resource(DeviceUnits, '/Devices/<int:id>/units')
-api.add_resource(TimeSeries, '/Devices/<int:id>/n=<int:n>')
-
-if __name__ == "__main__":
-    logging.basicConfig(format='%(asctime)s %(message)s', filename='example.log', level=logging.DEBUG)
+from fastapi import FastAPI
+from fastapi.responses import FileResponse
+from .Devices.Arduino import Arduino
+import logging
+app = FastAPI
```
Comment (OkeWoke) [inline web_logger/__init__.py:5]: app = FastAPI()

Comment (OkeWoke) [inline web_logger/__init__.py]:
```suggestion
app = FastAPI()

```

File: web_logger/__init__.py (hunk 2 above)
Comment (OkeWoke) [inline web_logger/__init__.py]:
```suggestion
def ListDeviceDims(device_id: int):

```

File: web_logger/__init__.py (hunk 1 above)
Comment (OkeWoke) [reply web_logger/__init__.py]: Have a look a this:
https://fastapi.tiangolo.com/advanced/templates/

**Comment by NicoEvZ** [comment]
Changes have been made, please review.

**Pull request merged**